src/gui/shell.h

```diff
@@ -181,6 +189,8 @@ private slots:
 	ShellOptions m_options;
 	PopupMenu m_pum;
 	bool m_mouseEnabled;
+    std::vector<CmdWidget*> m_cmdline_list;
```

Comment:
Didn't read the doc about that, but this seems like what I had in mind.

Reply:
I think a QGridLayout would be better there. I like the idea of having the context of the command when spawning "inner" command lines like this (in the screenshot the cursor is missing because I need to fix that, and the `"` is the `cmdline_special_char` which is not styled yet) : 

![screenshot_20190101_230013](https://user-images.githubusercontent.com/10496163/50576758-158c5380-0e19-11e9-887d-3205bf1224ba.png)